src/domains/plugin/pages/PluginView/__snapshots__/PluginView.test.tsx.snap

```diff
@@ -586,7 +586,7 @@ exports[`Plugin View should render plugin content with logo 1`] = `
       </div>
     </nav>
     <div
-      class="flex flex-col flex-1 mt-5"
+      class="flex flex-col flex-1"
     >
       <div
         class="bg-theme-background px-4 sm:px-6 lg:px-10 py-4 -mt-5"
```

Comment:
@goga-m can you remove this `-mt-5` from `PluginView`?

```suggestion
        class="bg-theme-background px-4 sm:px-6 lg:px-10 py-4"
```

Reply:
Sneak passed in conflict merge. Thanks, @luciorubeens